### PR TITLE
#4638 [Manager] Sort JSX props order via eslint

### PR DESCRIPTION
**Pull request by konekoya**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #4638)
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [x] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/package.json [hunk 1]
```diff
@@ -121,6 +121,14 @@
           "ignoreDestructuring": true
         }
       ],
+      "react/jsx-sort-props": [
+        "warn",
+        {
+          "shorthandLast": true,
+          "ignoreCase": false,
+          "callbacksLast": false
```
Comment (konekoya) [inline ohara-manager/client/package.json:129]: There are a couple of more options that we can customize with, would love to hear from you @oharastream/frontend :)

```
"react/jsx-sort-props": [<enabled>, {
  "callbacksLast": <boolean>,
  "shorthandFirst": <boolean>,
  "shorthandLast": <boolean>,
  "ignoreCase": <boolean>,
  "noSortAlphabetically": <boolean>,
  "reservedFirst": <boolean>|<array<string>>,
}]
```

File: ohara-manager/client/src/components/Autofill/AutofillEditor.js [hunk 2]
```diff
@@ -118,41 +118,41 @@ const AutofillEditor = props => {
...
-                autoFocus
                 component={InputField}
                 label="name"
                 margin="normal"
                 name="displayName"
-                required
                 type="text"
                 validate={composeValidators(required, duplicateDisplayName)}
+                autoFocus
+                required
```
Comment (eechih) [inline ohara-manager/client/src/components/Autofill/AutofillEditor.js:139]: 為何 autoFocus 排到後面呢？

File: ohara-manager/client/package.json (hunk 1 above)
Comment (eechih) [reply ohara-manager/client/package.json:129]: 我們可以把 shorthandLast 改為 false 嗎？

File: ohara-manager/client/src/components/Autofill/AutofillEditor.js (hunk 2 above)
Comment (konekoya) [reply ohara-manager/client/src/components/Autofill/AutofillEditor.js:139]: `shorthandLast` :)

should we just sort everything by alphabet?

File: ohara-manager/client/package.json (hunk 1 above)
Comment (konekoya) [reply ohara-manager/client/package.json:129]: Definitely :)

File: ohara-manager/client/src/components/Autofill/AutofillEditor.js (hunk 2 above)
Comment (eechih) [reply ohara-manager/client/src/components/Autofill/AutofillEditor.js:139]: 純屬個人的喜好，看大家的意見？

Comment (konekoya) [reply ohara-manager/client/src/components/Autofill/AutofillEditor.js]: I'm okay with this :)

File: ohara-manager/package.json [hunk 3]
```diff
@@ -13,11 +13,14 @@
     "start:prod": "cross-env NODE_ENV=production node start.js",
     "start:client": "yarn --cwd client start",
     "lint": "run-s lint:server lint:client",
+    "lint:fix": "run-s lint:server:fix lint:client:fix",
```
Comment (konekoya) [inline ohara-manager/package.json:16]: Added a npm script for fixing linter error if it's fixable

**Comment by konekoya** [comment]
retry

**Comment by konekoya** [comment]
retry manager-e2e

File: ohara-manager/client/src/components/Autofill/AutofillEditor.js (hunk 2 above)
Comment (konekoya) [reply ohara-manager/client/src/components/Autofill/AutofillEditor.js]: @saivirtue and @wu87988622 Thoughts before i merging this?

Comment (saivirtue) [reply ohara-manager/client/src/components/Autofill/AutofillEditor.js]: +1 to `shorthandLast: false`

**Comment by konekoya** [comment]
retry manager-e2e

**Pull request merged**